LayoutKit/LayoutArrangement.swift

```diff
@@ -150,9 +150,6 @@ extension View {
      It doesn't do anything of view is already a subview.
      */
     fileprivate func addSubview(_ view: View, maintainCoordinates: Bool) {
-        if view.superview == self {
```

Comment:
Why does this fix work? What was the bug? Is it worth adding some comments to this method saying what needs to stay in order?